docs/design/jet/025-jet-job-partition-pruning.md

```diff
@@ -123,9 +123,85 @@ try to choose only the required members.
 
 ### Scan partition pruning
 
-Partition pruning for IMap scan is a pretty simple optimization: we will extract the partition key 
-condition during SQL opt phase, pass it to a specialized processor supplier which will spawn 
-`ReadMapOrCacheP` with only required partitions to scan.
+Partition pruning for IMap scan supposed to be a pretty simple optimization: we should extract 
+the partition key condition(s) during SQL opt phase, pass it to a specialized processor meta supplier 
```

Comment:
"during SQL opt phase" - exactly when this happens (in planner or in postprocessing of the plan) hopefully will be described in SQL part of the TDD